snap/snapcraft.yaml

```diff
@@ -25,7 +25,7 @@ apps:
       LANG: C.UTF-8
       LANGUAGE: C.UTF-8
       LC_ALL: C.UTF-8
-      JAVA_HOME: $SNAP/usr/lib/jvm/default-java
+      JAVA_HOME: $SNAP/usr/lib/jvm/java-1.16.0-openjdk-amd64
```

Comment:
Please use $SNAP_ARCH instead of `amd64`

Reply:
@dcap0 - how about we add a symlink?

```
$SNAP/usr/lib/jvm/default-java -> $SNAP/usr/lib/jvm/java-1.16.0-openjdk-$SNAP_ARCH
```

Reply:
imo, changing the env variable is preferred over creating a symlink because it removes a level of indirection.